src/components/LottieLoader.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div ref="lottie" class="lottie-loader" :style="`width: ${size}px; height: ${size}px`">
+    <lottie-animation path="json/loader.json" />
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator'
+
+import LottieAnimation from 'lottie-vuejs/src/LottieAnimation.vue'
+import { delay } from '@/utils'
+
+@Component({
+  components: {
+    LottieAnimation
+  }
+})
+export default class LottieLoader extends Vue {
+  @Prop({ default: '', type: String }) readonly size!: string
+
+  async waitForLoadersMount (): Promise<void> {
+    await delay()
+    const lottieLoader = document.querySelector('.lottie-loader') as any
+    const loaders = document.querySelectorAll('.el-lottie-loading') as any
+    if (lottieLoader === null || loaders === null) {
+      return await this.waitForLoadersMount()
+    }
+    if (loaders.length) {
+      loaders.forEach(loader => {
+        loader.parentElement.appendChild(lottieLoader as Node)
+        loader.parentElement.removeChild(loader)
+      })
+    }
+  }
```

Comment:
I propose to make this component standalone, and use it without  `el-loading`, but instead.

```
<div class="loader-container"> <- wrapper for conditional content
  <div v-if="loading" class="lottie-loader" :style="`width: ${size}px; height: ${size}px`"> <- show loader if "loading" is true
   <lottie-animation path="json/loader.json" />
  </div>
  <slot v-else /> <- else show wrapped content
</div>
```